src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -121,13 +121,11 @@ class LinkView extends React.Component<ILinkViewProps, ILinkViewState> {
                         />
                     </div>
                     <div className={s.inputContainer}>
```

Comment:
`<InputContainer>` and className={s.inputContainer} seems confusing..

Reply:
Could we just have these dropdowns include the inputcontainer part inside them? With an optional label for the cases where we need "KUTSU-/JÄTTÖ-...."

Reply:
We could have label for dropdown container but then we should use labels for dropdowns as well

Reply:
Made a new card to implement this fix: https://github.com/HSLdevcom/jore-map-ui/issues/283

Reply:
Sounds good